coordinator/src/main/scala/com/socrata/datacoordinator/service/Service.scala

```diff
@@ -59,8 +60,7 @@ class Service(serviceConfig: ServiceConfig,
               datasetContents: (DatasetId, Option[String], CopySelector, Option[UserColumnIdSet],
                                 Option[Long], Option[Long], Precondition, Option[DateTime], Boolean) =>
                                datasetContentsFunc => Exporter.Result[Unit],
-
-              secondaries: Set[String],
+              common: SoQLCommon,
```

Comment:
Exposing SoQLCommon into Service seems like a bit of a violation of the current (somewhat strange) way that separation of concerns is handled in Service.  In the existing code Service has no notion of a transaction and more or less calls a single function for each that handles the entire database side of things with a single "universe" being created for each call.   

The simplest way to follow that pattern would be to have the function that is passed in be able to do the work that requires looking up the secondaries itself... then it could all be done with the same universe to keep the notion of "one transaction per call".  eg. for doGetSecondaryManifest have datasetsInStore have a way to return an indication that the store doesn't exist.  

That said... the whole way Main and Service interact seems a bit strange to begin with, and in this particular case isn't _actually_ that bad as far as opening multiple transactions, etc. so maybe it isn't worth worrying about.  On the other hand, I wouldn't want someone else to come along and start using common.universe to do more complicated operations such as update operations from within Service.